java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2129_java.html

```diff
@@ -25,5 +25,5 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p><code>BigDecimal</code> constructor with <code>double</code> argument is ignored as using <code>valueOf</code> instead might change resulting
-value. See {rule:java:S2111}.</p>
+value. See {rule:java:S2111} .</p>
```

Comment:
`S2111` is not converted into `{rule:java:S2111}` by rule-api if there's no space before the `.` :persevere: